azext_iot/constants.py

```diff
@@ -7,7 +7,7 @@
 
 import os
 
-VERSION = "0.13.1"
+VERSION = "255.255.14"
```

Comment:
Once this is ready for merge we'll have to conform to our standard release version semantics.

Reply:
Will need to rerun tests on 2/8

Reply:
changed to final values